rclpy/services/minimal_client/client_async.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+import rclpy
+
+from example_interfaces.srv import AddTwoInts
+
+
+def main(args=None):
+    rclpy.init(args)
+
+    node = rclpy.create_node('minimal_client_async')
+
+    cli = node.create_client(AddTwoInts, 'add_two_ints')
+
+    req = AddTwoInts.Request()
+    req.a = 41
+    req.b = 1
+    # TODO(mikaelarguedas) remove this once wait for service implemented
+    # wait for connection to be established
+    # (no wait for service in Python yet)
+    time.sleep(1)
+    cli.call(req)
+    while rclpy.ok():
+        if cli.response is not None:
```

Comment:
What do you think about this TODO message? https://github.com/ros2/examples/pull/140/commits/53060743ee85c5d602ff942fada4193284573e10